### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ogsevko**

### CODE REVIEW

File: src/getTime.js [hunk 1]
```diff
@@ -18,7 +18,14 @@
...
 function getTime(str) {
-  // write code here
+  const regexTime = /\b(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]\b/g;
+  const time = str.match(regexTime);
+
+  if (time === null) {
+    return '';
+  }
+
+  return time[0];
```
Comment (solaryasha) [inline src/getTime.js:28]: Dmytro told me that we can use a ternary operator in this task.

**Review by solaryasha — CHANGES_REQUESTED** [review]

File: src/getTime.js (hunk 1 above)
Comment (ogsevko) [reply src/getTime.js:28]: OK cool :)

**Review by solaryasha — APPROVED** [review]
Nice !

**Review by danheim — APPROVED** [review]
Nice

**Pull request closed without merging**